packages/eslint-config-airbnb-base/rules/style.js

```diff
@@ -303,7 +309,11 @@ module.exports = {
 
     // require or disallow space before function opening parenthesis
     // http://eslint.org/docs/rules/space-before-function-paren
-    'space-before-function-paren': ['error', { anonymous: 'always', named: 'never' }],
+    'space-before-function-paren': ['error', {
+      anonymous: 'always',
+      named: 'never',
+      asyncArrow: 'always'
```

Comment:
Even though the changed peer already is a breaking change? Along with import v2 already in master?